Added tip about custom plugin development (#82)
strapi/documentation: developer-docs/latest/plugin-development/quick-start.md

In a new terminal window:

1. Generate a new plugin: `cd /path/to/myDevelopmentProject && strapi generate:plugin my-plugin`

::: tip NOTE

> After you have succesfully generated your new custom plugin, you would need to run `strapi build` to generate the new field in the admin UI

Reviewer: Nice catch! That is indeed a necessarily step.

```suggestion
After you have successfully generated a plugin, you need to run `strapi build` which adds the new plugin to the admin panel. 
```
Or you could mention it as the second step.
